package.json

```diff
@@ -16,7 +16,7 @@
 		"refactoring"
 	],
 	"engines": {
-		"vscode": "^1.65.0"
+		"vscode": "^1.76.0"
```

Comment:
this is bang up-to-date from february- presumably it does mean we'll no longer support older versions of vs code. Probably that's fine since they just need to update. 

The old one was Feb 22 - this is Feb 23

Reply:
This field is advisory only, it will still work with older vscodes

https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines